### ISSUE TITLE
Offline capability in Mobile device Browser

### ISSUE THREAD

**Issue by nwaughachukwuma** [vuejs-templates/pwa#116]
Hi Team,

This awesome pwa application seem to have some limitations working offline on mobile browsers and raspberry pi chromium browser. It goes to blank screen and await internet availability before it fetches contents. This behavior is not present on the laptop and desktop browsers. What could be the problem plus is there a fix?

**Comment by jeffposnick** [comment]
Could you share some additional details?
- What public URL can we visit to reproduce the issue?
- What versions of Chromium did you test this with on the Raspberry Pi?
- What other mobile browsers did you test this with (including their versions)?

**Comment by nwaughachukwuma** [comment]
Hi @jeffposnick, I discovered that this is an issue with the browser version. My phone just updated it's web view and it's working fine now. But this is not the case on the raspberry pi, as it still goes to blank screen when the device is powered back on. I believe this is an issue with browser support for service worker. You can visit the site here: [dhis2-daily-reporting.firebaseapp.com/#/](https://dhis2-daily-reporting.firebaseapp.com/#/)

**Comment by jeffposnick** [comment]
I visited your web app, and it installs its service worker and loads cache-first when I tested from the current version of Chrome.

I'm not sure what version of Chromium you're testing on with the Raspberry Pi, but fundamentally, if you're using a browser version that doesn't support service workers, then you won't get the benefits of cache-first loading, and you'd see the behavior you describe.

As it stands, I don't think there's anything actionable here, at least not from the perspective of the Vue PWA template.
